src/pages/UserProfilePage.tsx

```diff
@@ -0,0 +1,62 @@
+import { useEffect, useState } from 'react';
+
+import * as userApi from "../api/userApi";
+
+interface UserProfile {
+  displayName: string;
+  email: string;
+}
+
+interface UserId {
+    id: string;
+}
+
+export default function UserProfilePage({id}: UserId) {
+    const [userProfile, setUserProfile] = useState<UserProfile>({displayName: "", email: ""});
+
+    useEffect(() => {
+        const fetchUser = async () => {
+            const user = await userApi.getUser(id);
+            setUserProfile({displayName: user.displayName, email: user.email});
+        }
+
+        fetchUser();
+    }, []);
+
+  const handlePasswordChange = () => {
+    // TODO: Implement password change functionality
```

Comment:
This is probably out of scope for this PR, but we do have the backend support for it. If you send a PATCH request to /users/<user-id> with a new password it'll update. We don't have a ticket for the integration, so you could do it here or we can create that separately (if we want to have another page for it now)

Reply:
Sounds good! I think I'll just leave it for another ticket so this one doesn't get too big https://gaming-backlog.atlassian.net/browse/GB-84